scripts/Interface/ui/drill/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(PYUNITTEST_QT_API pyqt5)
+
+set(TEST_PY_FILES
+  DrillTableWidgetTest.py
+  DrillModelTest.py
+  DrillPresenterTest.py
+  DrillViewTest.py
+  DrillTest.py
+  DrillParameterControllerTest.py
+  DrillSettingsDialogTest.py
```

Comment:
👍  Now all the tests are run with `ctest -R Drill`.

Reply:
:+1: